src/BinSkim.Driver/BinSkim.Driver.csproj

```diff
@@ -37,7 +37,7 @@
 
   <ItemGroup>
     <Reference Include="dia2lib">
-      <HintPath>..\..\refs\dia2lib.dll</HintPath>
+      <HintPath>..\..\refs\Dia2Lib.dll</HintPath>
```

Comment:
The name changed is needed because in Linux it is case sensitive.

Reply:
@shaopeng-gh What exactly is the issue when the SARIF SDK `Dia2Lib.dll` is used?

Reply:
misssing classes: IDiaEnumSegments, IDiaSectionContrib etc.
conflict classes: IDiaDataSource2, IDiaDataSource3
![image](https://github.com/microsoft/binskim/assets/81775155/609695ed-c6c5-46df-9336-aca15b30f996)

Reply:
If there isn't a reason *not* to change the Reference itself (as opposed to the hint), can you please change it to agree with the hint?

Reply:
Same comment for lines below.

Reply:
updated the 
`<Reference Include="dia2lib">`
to
`<Reference Include="Dia2Lib">`.
I have searched the whole repo there is 0 lower case of `dia2lib`.